.github/workflows/driver_test.yml

```diff
@@ -20,14 +20,26 @@ jobs:
     strategy:
       matrix:
         configurations: [Debug, Release]
-    runs-on: windows-latest
+
+    runs-on: [windows, self-hosted]
```

Comment:
Line 4 points to information on the github environment.  Does that apply to self-hosted?  If not, replace that comment with one that provides information on how self-hosted works.   Maybe point to your SelfHostedRunnerSetup.md doc?